### DISCUSSION TITLE
enable download of images created by geometry to texture op

### DISCUSSION THREAD

**Discussion by MarcVdH** [cables-gl/cables_docs#681] [Feature Requests] [answered]
is it possible to download 32 bit images created by the geometry to texture op?
![download image](https://github.com/cables-gl/cables_docs/assets/126583906/59f47416-02b8-48f3-91bf-51819e4b244e)

**Comment by pandrr** [comment]
no, there is no way currently

only way is to use the rgbe format, but that has its own limits

**Comment by pandrr** [comment, marked as answer]
the problem is, that its not a simple "enable".
there is no javascript way of doing this. a solution could be to use exr files.
but i dont know any js lib that writes them, implementing on our own is probably quite some work and not our priority now.

**Comment by MarcVdH** [comment]
thanks for answering. I am afraid I have no idea about the technical implications of my request(s).
sounds like this one requires a lot of work so I can fully understand it is not a current priority.
I still owe you a big thank you for fixing the issue with the zoom blur mask almost instantaneously. That was super sweet!
